charts/chronicle/templates/opa-init.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if .Values.opa.enabled }}
+{{$stlServiceName := include "lib.call-nested" (list . "sawtooth" "common.names.fullname")}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "common.names.fullname" . }}-opa-init
+  labels: {{ include "chronicle.labels" . | nindent 4 }}
+    component: opa-init
+spec:
+  backoffLimit: 5
+  template:
+    metadata:
+      labels: {{- include "chronicle.labels" . | nindent 8 }}
+        component: opa-init
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: {{ include "lib.serviceAccountName" . }}
+      automountServiceAccountToken: true
+      affinity: {{ include "lib.safeToYaml" .Values.affinity | nindent 8 }}
+      containers:
+        - name: opa-init
+          {{- include "lib.image" (dict "imageRoot" .Values.opa.opaInit.image "global" .Values.global ) | nindent 10 }}
+          command: [ "bash", "-c"]
+          args:
+            - -exc
+            - |
+              HOST={{ $stlServiceName }}.{{ .Release.Namespace }}.svc.cluster.local
+              PORT={{ include "chronicle.sawtooth.sawcomp" . }}
+
+              /script/wait-for-it.sh $HOST:$PORT --timeout=0
+
+              cd /tmp
+              /usr/local/bin/opactl generate --output ./root.pem
```

Comment:
You are generating the root key here in the `tmp` directory, and worse in the tmp dir of a container - that means it is doubly going to be thrown away at the end of this job never to be recovered.  That is definitely not what you want.   

You need to upload that root.pem into a secret - which also means you need to check that this secret does or does not exist already before doing any of this init job.